mdpow/run.py

```diff
@@ -71,7 +73,7 @@ def setupMD(S, protocol, cfg):
 
 def get_mdp_files(cfg, protocols):
     """Get file names of MDP files from *cfg* for all *protocols*"""
-    import ConfigParser
+    import configparser
```

Comment:
Import at top of file. Check if this works in Python 2.7 or if you need the try/except for the import as in my earlier comment.

Reply:
And yes, I know it's not your fault that the import was in a function.

Reply:
use six.moves